lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1412,6 +1412,18 @@ Value *SPIRVToLLVM::oclTransConstantPipeStorage(
                             GlobalValue::NotThreadLocal, SPIRAS_Global);
 }
 
+static void replaceOperandWithIntrinsicCallResult(Value *&V) {
```

Comment:
```suggestion
static void replaceOperandWithAnnotationIntrinsicCallResult(Value *&V) {
```

Please also add a comment about the purpose of this function, because it relies on some particular LLVM IR representation which is only used for FPGA-specific extensions.